### PR TITLE
feat: Customer list overview

### PR DESCRIPTION
**Pull request by olivermrbl**
**What**
Adds Customer list overview
Adds `TablePagination` component in table

**How**
`react-table` PoC  implementation

### CODE REVIEW

File: src/domain/customers/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          </div>
+        </BodyCard>
+      </div>
+    </div>
+  )
+}
+
+const Customers = () => {
+  return (
+    <Router>
```
Comment (pKorsholm) [inline src/domain/customers/index.tsx:32]:
```suggestion
    <Router className="h-full" >
```

**Review by pKorsholm — APPROVED** [review]
This looks great 😄  I have only a few comments i've left as suggestions

File: src/domain/customers/index.tsx (hunk 1 above)
Comment (olivermrbl) [reply src/domain/customers/index.tsx:32]: Adding this class results in the following table view. I ran it by Ludvig and he would rather have the "shrinked" table so he's gonna update the table designs accordingly 👍 
image

Comment (pKorsholm) [reply src/domain/customers/index.tsx:32]: cool 👍  well then just ignore the comment 😄

**Pull request merged**